ci/build.py

```diff
@@ -297,22 +377,24 @@ def script_name() -> str:
                         type=str)
 
     args = parser.parse_args()
+
     def use_cache():
-        return args.cache or under_ci()
+        return not args.no_cache or under_ci()
 
     command = list(chain(*args.command))
     docker_binary = get_docker_binary(args.nvidiadocker)
     shared_memory_size = args.shared_memory_size
     num_docker_build_retires = args.docker_build_retries
 
     if args.list:
-        list_platforms()
+        print(list_platforms())
```

Comment:
You used logger before, maybe use it here as well instead of print?

Reply:
Is different concerns, this prints unconditionally to stdout which is what you want. Logging can be supressed. This is intentional